src/mixins/itext_key_behavior.mixin.js

```diff
@@ -51,6 +51,7 @@ fabric.util.object.extend(fabric.IText.prototype, /** @lends fabric.IText.protot
 
   onClick: function() {
     // No need to trigger click event here, focus is enough to have the keyboard appear on Android
+    // 2015-02-05 with forced focus on enterEditing maybe we can remove this now.
```

Comment:
Yes i think you are right,
i do not even like that onClick event is on_key_behaviour instead that click_behaviour. I remove it.